### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #13 [July] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#9662] [Wallet and (d)App Council]
# Monthly Celo Wallet and (d)App Council

##  📆 Quick links 

- When: **Wednesday, July 20 (9am PDT/6pm CEST)**
- Where: **[Google Meet](https://meet.google.com/uwj-wfse-uzm)**
- Agenda: **comments**👇
- Previous: https://github.com/celo-org/celo-monorepo/discussions/9625
- Recording: [Google Drive](https://drive.google.com/file/d/13RHRDxKU09_sR2_-Akhr8GBT20zaUSaR/view?usp=sharing)

## 🤷‍♂️ What is this?

A monthly meeting for Celo **developers, founders,** **product** people (anyone really!) to come together and
- find consensus (coordinate),
- learn about needs, pain points, and desires
- present and collect technical feedback

## 🗒️  Agenda

- Comment with a short sentence describing what you'd like to discuss
- Don't worry, it can be small or big! 
- Every builder is welcome (and we are very easy going, I promise 😄)
- Upvote topics you're interested in

## Housekeeping

_You can safely ignore this!_

- [x] Send update calendar invite with Github link
- [x] Encourage additions to the agenda
- [x] Publicise on forum and Discord
- [ ] RECORD meeting 🎥

**Comment by product-matt** [comment]
(short) stCELO updates: WebApp, Mobi Pools, and moar!

**Reply by arthurgousset** [reply]
**Meeting notes**:
- close to finished with webapp to use stCELO
- Mobius is providing a stCELO-CELO pool
- Evan (Ubeswap): Could you elaborate on the 3 day unlocking period?
  - Can't avoid 3 day unlocking period but can trade faster (without price guarantee) if a secondary market exists.
- Patrick (Moola): Could you speak to the validator election process
  - Process is documented here: https://forum.celo.org/t/stcelo-launch-on-friday-feedback-on-our-plan-for-validator-election/3897
- Patrick (Moola): Do the 8 validators rotate?
  - No, because of challenges with withdrawing stCELO and revoking votes.
- Gonza (Celo Tracker): Does the protocol take any fees
  - No.

**Comment by aaronmgdr** [comment]
Vibe check: Proposition for react-celo replacing web3 + contractkit with ethersjs + celo-ethers-wrapper.

**Reply by arthurgousset** [reply]
**Meeting notes**: 
- Rossy is in favour
  - ethersjs + celo-ethers-wrapper is what Celo wallet has always used to keep it small and fast
  - celo-ethers-wrapper (Rossy wrote) came first and supports Celo quirks (like `feeCurrency`)
  - https://www.npmjs.com/package/@abacus-network/celo-ethers-provider
  - ethersjs library is seemingly outpacing web3
- Bernardo is in favour too
  - ImpactMarket doesn't use CK

**Comment by ssubram11** [comment]
Vibe check: Are off-ramps of interest for folks in this call? If yes, what kind of off-ramps are your users requesting?

**Reply by arthurgousset** [reply]
**Meeting notes**:
- Works on payments and applications team
- Would love some feedback on what type of off-ramps might be useful to participants

**Off-ramps**:
- Bernardo (Impact Market): 
  - off-ramps are mainly interesting for merchants
  - trying not to incentivise users to off-ramp 
- Patrick (Moola):
  - cUSD - USD off-ramps are very important
- Charlie (Valora):
  - geographical coverage matters to our users (having as many locations as possible)
  - we use fiat connect for that

**On-ramps**:
- Bernardo (Impact Market): 
  - We'd be interested to support direct donations on our website using regular debit/credit cards
  - On-ramp from fiat to ImpactMarket smart contract (instead of into user account).
- Charlie (Valora)
  - I heard native integration are more effective than web views for on-ramps
  - On-ramps are mission critical for us.
- Silas (Valora):
  - > one nice thing about native integrations is that is facilitates funnel optimizations because the application (e.g., Valora) has access to the analytics and can appropriately tune the experience to users.  this is much harder to do with webviews.
- Evan (Ubeswap):
  - on-ramps are crucial to the success of Celo
  - limit order protocol could be an interesting primitive for CICO.
  - foundation makes large cUSD limit sell order for anyone coming with USDC.
- Patrick (Moola):
  - This is a great idea 👆

**Comment by arthurgousset** [comment]
📽 Here is the [recording](https://drive.google.com/file/d/13RHRDxKU09_sR2_-Akhr8GBT20zaUSaR/view?usp=sharing) for anyone that missed the meeting.
